### PR TITLE
Adding Saurabh as contributor

### LINKED ISSUE #173: Add Saurabh Thankur as a community member
****Current Behavior**
@thakursaurabh1998 is a new community contributor.

**Desired Behavior**
A profile for @thakursaurabh1998 should be created and visible on `/community`.**

### PR DESCRIPTION
**Pull request by willcalcote**
Fixes #173 
Signed-off-by: Will Calcote <wmcalcote@gmail.com>

### CODE REVIEW

File: collections/_contributors/saurabh-thakur.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: person
+name: Saurabh Thakur
+position: Contributor
+image_path: /images/photos/Saurabh Thakur.jpg
```
Comment (thakursaurabh1998) [inline collections/_contributors/saurabh-thakur.md:5]: Shouldn't this be `/images/photos/saurabh-thakur.jpg` ??

File: collections/_contributors/saurabh-thakur.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+twitter: thakursaurabh98
+github: thakursaurabh1998
+blurb: Mesh Head
```
Comment (thakursaurabh1998) [inline collections/_contributors/saurabh-thakur.md:8]: Blurb can simply be `Contributor`

Comment (ashiscs) [reply collections/_contributors/saurabh-thakur.md:8]: That is default set to Mesh Head for every contributor.
What you are talking about is defined in position..

Comment (thakursaurabh1998) [reply collections/_contributors/saurabh-thakur.md:8]: Ohh yeah sorry!

**Comment by leecalcote** [comment]
@willcalcote it looks like you'll need to fix the file path used to point to Saurobh's profile picture.

**Comment by leecalcote** [comment]
💥 @thakursaurabh1998 😀

**Pull request merged**